### DISCUSSION TITLE
dreamlte-ricedroidOSS-thirteen-ayeneo please help

### DISCUSSION THREAD

**Discussion by ayeneo** [ROM-builders/temporary#19576] [Q&A] [answered]
![Screenshot_20230403-190409_Chrome](https://user-images.githubusercontent.com/129434536/229566781-883a4a7e-899c-473c-a54d-41533655bd26.png)

**Comment by Apon77** [comment, marked as answer]
build link?

**Reply by ayeneo** [reply]
https://github.com/ROM-builders/temporary/pull/19574

**Reply by Apon77** [reply]
Your repo init line is wrong. Error in your photo is already saying that. Please look carefully
